models/image.py

```diff
@@ -788,8 +806,8 @@ def from_images(cls, images, index=0):
         output.end_mjd = max([image.end_mjd for image in images])  # exposure ends are not necessarily sorted
 
         # TODO: what about the header? should we combine them somehow?
+        output.info = images[index].info
```

Comment:
We should probably do some filtering on info and header just like you did with the attributes.

Practically speaking, when we swarp the images together, it will strip all of the header information out; you can pass an argument to swarp telling it which header keywords to copy from the first image in the list.  We should update the image's header and info attributes after the swarp is done.  (Not sure if it does that right now, will have to look.)

Reply:
I have issue #172 to think about and sort this out.